### PR TITLE
Document a workaround to RuntimeException 'boom' [skip ci]

### LINKED ISSUE #7767: [BUG] VS Code / Metals / Bloop integration fails with java.lang.RuntimeException: 'boom'
**Describe the bug**
```
ERROR Unexpected error when compiling rapids-4-spark-sql_2.12: 'java.lang.RuntimeException: boom'
```
**Steps/Code to reproduce bug**

Create Bloop build targets
```
mvn clean install ch.epfl.scala:maven-bloop_2.13:bloopInstall -pl tests -am -Dskip -DskipTests -Dmaven.javadoc.skip -Dbuildver=331 -DdownloadSources=true
```

Start Metals and observe in Metals output log:
```
2023.02.13 21:53:27 ERROR Unexpected error when compiling rapids-4-spark-sql_2.12: 'java.lang.RuntimeException: boom'
```

Connecting to Bloop server from the CLI client reveals a more clear stacktrace:
```
Caused by: java.lang.RuntimeException: boom
        at com.sourcegraph.semanticdb_javac.SemanticdbTaskListener.finished(SemanticdbTaskListener.java:76)
        at com.sun.tools.javac.api.ClientCodeWrapper$WrappedTaskListener.finished(ClientCodeWrapper.java:681)
        at com.sun.tools.javac.api.MultiTaskListener.finished(MultiTaskListener.java:111)
        at com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1342)
        at com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1296)
        at com.sun.tools.javac.main.JavaCompiler.compile2(JavaCompiler.java:901)
        at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:860)
        at com.sun.tools.javac.main.Main.compile(Main.java:523)
        ... 24 more
Caused by: com.sourcegraph.semanticdb_javac.CompilationUnitException: file:/home/user/gits/NVIDIA/spark-rapids/sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnarCopyHelper.java
Caused by: com.sun.tools.javac.code.Symbol$CompletionFailure: class file for org.apache.spark.sql.types.Decimal$1 not found
```

https://github.com/sourcegraph/scip-java/blob/b7d268233f1a303f66b6d9804a68f64b1e5d7032/semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbTaskListener.java#L76

**Expected behavior**

Metals should continue to work 

**Environment details (please complete the following information)**
 - Environment location: local dev

### PR DESCRIPTION
**Pull request by gerashegalov**
Fixes #7767

in addition update bloopInstall usage to the new Maven coordinates per https://github.com/scalacenter/bloop-maven-plugin/blob/c4d7f0f34cb7ad4a992abace25e3ef12cf08b086/README.md?plain=1#L14

### CODE REVIEW

**Comment by gerashegalov** [comment]
build

**Comment by gerashegalov** [comment]
verified locally with 331, and on Databricks with 330db

**Comment by gerashegalov** [comment]
build

File: build/buildall [hunk 1]
```diff
@@ -19,8 +19,6 @@ set -e
```
Comment (jlowe) [inline build/buildall:19]: 2023 copyrights

**Review by jlowe — APPROVED** [review]

**Comment by gerashegalov** [comment]
build

**Pull request merged**